app/assets/stylesheets/cypress/_utilities.scss

```diff
@@ -26,4 +26,17 @@ a.incomplete {color : #333}
   cursor: pointer;
   background: url(/assets/closed.gif) 4px 9px no-repeat;
   &.open {background:url(/assets/open.gif) 4px 9px no-repeat;}
-}
\ No newline at end of file
+}
+
+// Only display content to screen readers
+// From Bootstrap 3 & http://a11yproject.com/posts/how-to-hide-content/
+.sr-only {
```

Comment:
Properties should be ordered border, clip, height, margin, overflow, padding, position, width